Blueprints/BlueprintDefinitions/Msbuild-NETCore_2_1/EmptyFunctionDepInjection/template/src/BlueprintBaseName.1/DependencyResolver.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace BlueprintBaseName._1
+{
+    public class DependencyResolver
```

Comment:
I think I could eliminate `DependencyResolver` and move the `ServiceCollection` code to the `Function` ctor.  Would simplify things. @slang25 What do you think?